src/plugins/saved_objects_management/public/management_section/objects_table/components/header.test.tsx

```diff
@@ -38,12 +38,51 @@ describe('Header', () => {
       onExportAll: () => {},
       onImport: () => {},
       onRefresh: () => {},
-      totalCount: 4,
+      onDuplicate: () => {},
+      title: 'Saved Objects',
```

Comment:
title is not a required props for header I think.

Reply:
Have deleted the useless code.